lib/tasks/gather.js

```diff
@@ -736,6 +736,16 @@ class TaskGather extends SttTask {
       this.logger.info({evt}, 'TaskGather:_onTranscription - got empty transcript, continue listening');
       return;
     }
+    const confidence = evt.alternatives[0].confidence;
+    const minConfidence = this.data?.recognizer?.minConfidence;
```

Comment:
minor point, but please remove ? after this.data - there is always a 'this.data' object

Reply:
Remove ? after this.data